### DISCUSSION TITLE
Retina, 13", Ende 2012 with Sonoma 14.1.2 (OpenCore-Patcher)

### DISCUSSION THREAD

**Discussion by purekali** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1270] [Q&A]
Will it work on a **MacBook Pro Retina, 13", End 2012** with **Sonoma 14.1.2 (OpenCore-Patcher)**

I tried and failed. Any suggestions?

**Comment by MatthiasKerbl** [comment]
Hi @mymacteen,

Unfortunately, your MacBook model is not officially supported by AlDente. However, you can try this old version of AlDente Classic at your own risk: https://github.com/AppHouseKitchen/AlDente-Charge-Limiter/releases/tag/1.2

Best,
Matthias
